proposals/0000-create-asset-instruction.md

```diff
@@ -0,0 +1,60 @@
+---
+Title: CreateAsset Instruction
+Number: 0
+Status: Draft
+Version: 0
+Authors:
+ - vstam1
+Created: 2023-06-09
+Impact: Low
+Requires:
+Replaces:
+---
+
+## Summary
+
+The proposed change is to introduce a new Instruction called `CreateAsset`. The instruction allows foreign chains to create fully-backed derivatives. Currently, these derivatives are created using the `Transact` instruction, or directly on the sovereign chain. 
+
+
+## Motivation
+
+Currently, the `Transact` instruction is used to create derivatives. The `Transact` instruction requires knowledge of the foreign system to determine which call to make. The call can differ per system, and in FRAME-based systems, it can depend on different instances of a pallet. The CreateAsset instruction abstracts away the need to know this call. Sending consensus systems can send this instruction to any chain without having to know the structure of the destination chain. 
+
+## Specification
+
+The instruction would have the following specification:
+
+```rust
+CreateAsset {asset: MultiAsset, owner: MultiLocation}
```

Comment:
I would instead call this `AcquireAssetId`, since:

a) you're not actually *creating* any assets; and
b) you're appointing a controller for the given asset class/ID.

Alternatives are:

- `OccupyAssetId`
- `SeizeAssetId`
- `TakeAssetId`
- `AppointAssetIdAdmin`
- `ProcureAssetId`
- `ObtainAssetId`

Reply:
No - as I say you're not actually creating anything.